javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,131 @@
+"use strict"
+let personName;//declared here for global scope.
+const todoArray = [];
+
+function getReply(command){
+    if(typeof(command)!= "string"){
+     return 'please enter a valid command'       
+    }
+const arrayOfCommand = command.split(' '); 
+const todo = command.substring(4,command.length-11); // extracting the characters for pushing todoactivity 
+const firstNumber = Number(arrayOfCommand[2]); //converting into numbers for mathematical operation
+const secondNumber = Number(arrayOfCommand[4]);
+
+
+     switch (command){
+         case 'Hello my name is Rajani Busani':
```

Comment:
What you did is correct. It can be made more dynamic or an alternative can be like below example
 const commandWords = inputString.split(' ');

  if (inputString.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }

Reply:
thank you for letting me know about alternative startsWith()...